tests/ui/specs/VirtualList/VirtualGridList/With5-way/VirtualGridList-GT-28449-specs.js

```diff
@@ -0,0 +1,54 @@
+const {expectFocusedItem} = require('../../VirtualList-utils');
+const Page = require('../VirtualGridListPage');
+
+describe('Page scroll via 5-way', function () {
+	beforeEach(function () {
+		Page.open();
+	});
+
+	it('should scroll via 5-way on last item on the page [GT-28449]', function () {
+		const initVerticalScrollThumbPosition = Page.scrollThumbPosition();
+		// Step 3: Position the pointer on the last item in a current page.
+		Page.item(Number(Page.bottomRightVisibleItemId().slice(4))).moveTo();
+		Page.spotlightSelect();
+		// Step 3 Verify: Spotlight is on the item.
+		expectFocusedItem(14);
+		// Step 4: 5-way Down to the item below the last item on the current page.
+		Page.spotlightDown();
+		Page.delay(500);
+		// Step 4-1 Verify: The list Scrolled Up
+		const curVerticalScrollThumbPosition = Page.scrollThumbPosition();
+		expect(curVerticalScrollThumbPosition > initVerticalScrollThumbPosition).to.be.true();
```

Comment:
Please use to.be.above

Reply:
Same above link(https://github.com/enactjs/sandstone/pull/881#discussion_r620740599).